kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/OnlineShopUser.java

```diff
@@ -0,0 +1,25 @@
+package com.kodilla.good.patterns.challenges;
+
+public class OnlineShopUser implements Customer
+{
+    private final String firstName;
+    private final String lastName;
+
+    public OnlineShopUser (final String firstName, final String lastName)
+    {
+        this.firstName = firstName;
+        this.lastName = lastName;
+    }
+
+    @Override
+    public String getFirstName()
+    {
+        return this.firstName;
+    }
+    @Override
+    public String getLastName()
+    {
+        return this.lastName;
```

Comment:
w getterach powyżej nie używasz this. explicit

Reply:
Zgodnie z rozmową użycie zamierzone.